crates/config/src/lib.rs

```diff
@@ -34,6 +33,10 @@ use foundry_compilers::{
     error::SolcError,
     ConfigurableArtifacts, Project, ProjectPathsConfig,
 };
+
+// cyfrin
+pub use foundry_compilers::artifacts::remappings::{RelativeRemapping, Remapping};
```

Comment:
Publicizing this version of RelativeRemaping so that aderyn can use it instead of our own.